src/Suppressor.jl

```diff
@@ -109,4 +136,30 @@ macro capture_err(block)
     end
 end
 
+"""
+    @color_output enabled::Bool expr
+
+Enable or disable color printing for the given expression. Often useful in
+combination with the `@capture_*` macros:
+
+## Example
+
+@color_output false begin
+    output = @capture_err begin
+        warn("should get captured, not printed")
+    end
+end
+@test output == "WARNING: should get captured, not printed\n"
+"""
+macro color_output(enabled, block)
```

Comment:
Should signature be `macro color_output(enabled::Bool, block)` too?

Reply:
yeah, probably a good idea. fixed.